Feature/dashboard

nicholas-myers opened the pull request:
A user can view a Dashboard. The skeleton Components for the Dashboard view. PageHeader, Layout, Button used from Ant components. Styles set up in less and theme files see Readme in styles folder for instructions. Routes setup. Removed Skeleton code in RenderHomePage and set up the Ant Component using Layout.

## Before making the Pull Request:

- [x] Is there a description to this pull request that points to a specific user story, feature, or bugfix?
- [x] Are there comments where the code may be unclear?
- [x] Is all dead code, commented out code, etc. removed?
- [ ] Are there tests written for these changes?
- [x] Does the ReadMe need to be updated?


## Before being merged by the TPL:

- [x] Has the pull request been reviewed by at least two team members? 
- [x] Did they leave any comments or suggestions? 
- [x] Were those suggestions acted on?
- [x] Are all tests passing?
- [ ] Has the documentation been updated?

Commit:
Changed the site title to Apollo

Commit:
created Nav component file in home component, built basic code structure in react

Commit:
imported navigation component into home render page, for visual testing

Commit:
created the index.css file, imported into index.js

Commit:
removed test links and examples from home render page

Commit:
imported google fonts Kufam and red rose

Commit:
passed username through props to Navigation component

Commit:
had compile error, deleted repo and recloned to local, pull from the feature nav branch, deleted custom css and Nav component to now use the ant component library as custom css and component did not correctly apply styles

Commit:
updated the heading color to tan, removed Layout Component- Imported PageHeader Component into Homepage and changed background color for PageHeader component, Added Buttons to Header

Commit:
added Sider for Topics view, added Sider for Thread View, Content Section will hold Surveys

Commit:
renamed home.less to reset put in general styling reset preferences then imported to index.js

Commit:
fixed psoitioning of Pageheader

Commit:
added index less file to import all other less files into

Commit:
added styling to nav buttons

Commit:
added styling to Topics Button

Commit:
chenged Test to search for Apollo title, and added bottom border to header

Commit:
added a hello message to the username as subtitle

Commit:
changed the subtitle color by changing color in theme override

Commit:
changed branch name to correct feature name, deleted unused nav less file

Commit:
changed color for subtitle

Commit:
created the readme for styles with some general instructions for adding styles

Commit:
updated the readme to include more detailed instructions

Commit:
added Links and Routes to Button navigation

Commit:
changed button text color

Commit:
testing style changes

Commit:
installed dependency for atn-desig/icons, imported the User icon into Hompage, style user icon

Commit:
fixed import issue

Commit:
updated README

Commit:
updated README

mcelroyian committed:
Merge branch 'main' into feature/dashboard

mcelroyian reviewed (dismissed):
Looks good. I love that you included a test and readme information

mcelroyian committed:
Update RenderHomepage

I fixed merge conflict, but missed an extra import that was causing tests to fail. Fixed

mcelroyian reviewed (dismissed):
fixed

jiayi-ren reviewed (dismissed):
Looks great

mcelroyian committed:
added keys to components to attempt to fix errors

mcelroyian committed:
fixed broken tests, i changed how the logout works just to get tests runnign

mcelroyian reviewed (approved):
I broke the tests when i resolved merge conflict on homepage. I changed how logout worked to get tests passing again

jiayi-ren reviewed (approved):
👍

nicholas-myers commented:
yeah the Routes are dummy to the example stuff for now until they get replaced with the correct components

jslohner reviewed (approved):
Looks great man 👍
